pkg/odo/cli/component/create.go

```diff
@@ -777,23 +769,22 @@ func (co *CreateOptions) downloadProject() error {
 // Run has the logic to perform the required actions as part of command
 func (co *CreateOptions) Run() (err error) {
 	if experimental.IsExperimentalModeEnabled() {
-		// Download devfile.yaml file and create env.yaml file
+		// Download devfile.yaml file and create env.yaml file only if its not already present
 		if co.devfileMetadata.devfileSupport {
-			if !util.CheckPathExists(DevfilePath) {
```

Comment:
this check is not needed because we are already checking it above.

Reply:
This is needed as interactive mode uses devfile directly, remove it can make interactive mode download devfile from registry, which causes component type conflict issue.

Reply:
done